src/main/main.ts

```diff
@@ -410,6 +412,16 @@ ipcMain.on('settingsWindow', (event, args) => {
     event.returnValue = getAvailableDisplays();
     return;
   }
+
+  if (args[0] === 'getObsAvailableResolutions') {
+    if (!recorder) {
```

Comment:
pedantic comment, but perhaps neater as:

```
if (args[0] === 'getObsAvailableResolutions') {
  event.returnValue = { 'Base': [], 'Output': [] };
  if (!recorder) return;
  event.returnValue = getObsResolutions();
}
```

Reply:
I'm not a huge fan of non-braces if's, but I see you've done it in _many_ places so as you wish =)

Reply:
This was apparently not possible, as setting `event.returnValue` twice resulted in the second assignment being ignored and not returned. I suspect that behind the scens, any assignment to `returnValue` will instantly return the `ipc.sendSync()` call and thus render subsequent assignments pointless.

Reply:
To be honest I'm not sure what I prefer. I spent the first year or two of my career just writing Java as a {} using langauge and the coding standards at work mandate

```
if ()
{
 ...
}
else 
{
  ...
}
```

formatting. Not going to get excited about whatever goes in here.

Reply:
Interesting re the event.returnValue stuff though - I think I actually spotted that once too but obviously forgot. Fine as is!

Reply:
My person preference is that whatever makes the intent of the code easier to read is whatever I go with, but I can see how the following can look off, as the `else` could be easily missed.

```js
if (...) {
} else
if (...) {
}
```